rs/execution_environment/src/scheduler.rs

```diff
@@ -756,6 +757,8 @@ impl SchedulerImpl {
             );
             state.put_canister_states(canisters);
 
+            round_schedule.finish_inner_round(&mut state.canister_states, executed_priority_ids);
```

Comment:
I'm always confused about the terminology here, but isn't an "inner round" just the second part of the "outer round"? Isn't what we're finishing here an "[inner round] iteration"?

Reply:
Renamed to `finish_inner_round_iteration`